components/mpas-albany-landice/bld/namelist_files/namelist_defaults_mali.xml

```diff
@@ -49,8 +55,11 @@
 <config_damage_stiffness_min>0.1</config_damage_stiffness_min>
 <config_damage_rheology_coupling>.false.</config_damage_rheology_coupling>
 <config_damage_gl_setting>'nye'</config_damage_gl_setting>
-<config_damage_calving_method>'threshold'</config_damage_calving_method>
+<config_damage_calving_method>'none'</config_damage_calving_method>
```

Comment:
@trhille - I just want to make sure this is the correct new default. The automated scripts to reflect Registry changes in the bld files tend to default to the previous E3SM settings but this one appears to be set as part of this PR

Reply:
@jonbob, yes this is the correct new default. Thanks for checking!